env.yml

```diff
@@ -1,12 +1,14 @@
 name: payu
 channels:
   - accessnri
-  - coecms
   - conda-forge
+  - coecms
 dependencies:
+  - python==3.10
   # note that the deployment requires a specific version of 'accessnri::payu' to run
-  - accessnri::payu==1.1.3
-  - coecms::f90nml
+  - accessnri::payu==1.1.4
+  - coecms::yamanifest
```

Comment:
If we don't have an issue to deploy this to our own `conda` channel can we please make one? Thanks!

Reply:
Background: coecms is shuttering at the end of the year.

Reply:
Sure thing, issue created here #36